src/qp/metrics/point_estimate_metric_classes.py

```diff
@@ -170,3 +270,22 @@ def evaluate(self, estimate, reference):
         ez = (estimate - reference) / (1.0 + reference)
         mad = np.median(np.fabs(ez - np.median(ez)))
         return mad * SCALE_FACTOR
+
+    def compute_from_digest(self, digest):
+        SCALE_FACTOR = 1.4826
+
+        # calculation of `np.median(np.fabs(ez - np.median(ez)))` as suggested by Eric Charles
+        this_median = digest.inverse_cdf(0.50)
+        this_min = digest.inverse_cdf(0)
+        this_max = digest.inverse_cdf(1)
+        bins = np.linspace(this_min, this_max, self._num_bins)
+        this_pdf = digest.cdf(bins[1:]) - digest.cdf(bins[0:-1]) # len(this_pdf) = lots_of_bins - 1
+        bin_dist = np.fabs(bins - this_median) # get the distance to the center for each bin in the hist
+
+        sorted_bins_dist_idx = np.argsort(bin_dist) # sort the bins by dist to median
+        sorted_bins_dist = bin_dist[sorted_bins_dist_idx] # get the sorted distances
+        cumulative_sorted = this_pdf[sorted_bins_dist_idx[0:-1]].cumsum() # the cumulate PDF within the nearest bins
```

Comment:
@eacharles could I get a cross check on this line? In particular the indexing of `sorted_bins_dist_idx[0:-1]`.

I did this because `this_pdf` has length = `self._num_bins - 1` and `sorted_bins_dist_idx` has length = `self._num_bins`.

Long story short, I'm not sure if there was a better way to ensure that lengths of the arrays matched, or, given the current solution, if it makes more sense to drop the last element of `sorted_bins_dist_idx` or the first element.

Reply:
I think that dropping the last bin here is fine, as the only thing you are doing with this array is np.searchsorted to get the median.

Reply:
Quick comment here, since `sorted_bins_dist_idx` is a collection of indicies, excluding the last one in the list doesn't necessarily exclude the index that reads off the end of the `this_pdf` list. 

I've changed the location of the `[0:-1]` indexing so that we avoid reading off the end of `this_pdf`.

Reply:
@sschmidt23 w.r.t. your comment, I don't think I have the expertise to answer that question. For what it's worth, the pytdigest documentation says this about the `cdf` method: "Calculate approximate values of empirical cumulative distribution function (CDF) at given points."

Long story short, I'm not sure if that implies that it would introduce a half-bin bias or not.

Reply:
I think you could get the bin_centers with   bin_cents = (bins[0:-1] + bins[1:]) and then do the sorting on bin_cents, and that would issues with the pdf array being one shorter than the cdf array.

Reply:
Ok, I believe I followed. Please check the code again and see if I've interpreted this correctly.